IbisHardwareIGSIO/configio.cpp

```diff
@@ -29,8 +33,9 @@ void ToolConfig::Serialize( Serializer * ser )
 {
     ::Serialize( ser, "ToolName", ToolName );
     ::Serialize( ser, "ToolType", ToolType );
-    ::Serialize( ser, "ToolModelFile", ToolModelFile);
-    ::Serialize(ser, "ToolParamFile", ToolParamFile);
+    ::Serialize( ser, "ToolModelFile", ToolModelFile );
+    ::Serialize( ser, "ToolParamFile", ToolParamFile );
+    ::Serialize( ser, "FlipYAxis", FlipYAxis );
```

Comment:
Why does  the hardware module need to know that. Shouldn't we flip the image inside OpenIGTLinkIO and simplify life for the programmer?

Reply:
I thought it was easiest to do it on the client side and specify it in the config file, but it could surely be done differently. You mean you would add the flipping to the OpenIGTLinkIO connector? And pass the parameter to it in the constructor?